docs/schema.md

```diff
@@ -157,11 +157,11 @@ The defined database prefixes and their "home" databases are:
   </thead>
   <tbody>
     <tr>
-      <td><code>A</code></td>
-      <td><a href="https://storage.googleapis.com/android-osv/">Android Vulnerability Database</a></td>
+      <td><code>`ASB-A`/`PUB-A`</code></td>
+      <td><a href="https://source.android.com/docs/security/bulletin">Android Security Bulletin</a></td>
       <td>
         <ul>
-          <li>How to contribute: TBD</li>
+          <li>How to contribute: <a href="https://bughunters.google.com/about/rules/android-friends/6171833274204160/android-and-google-devices-security-reward-program-rules">Android Vulnerability Rewards Program</a></li>
           <li>Source URL: <code>N/A</code></li>
```

Comment:
Can you use `https://storage.googleapis.com/android-osv/<ID>.json` here?

Reply:
What do you think about putting just "https://storage.googleapis.com/android-osv" here? People can get a list of all the vulns and can drill down if they want to (we also have the "OSV Formatted URL" field right below with individual vuln URL format).

Reply:
Every other source here has <ID> in it by convention, so it would be  better to be consistent here.

Ideally, this URL would be for the URL that points to some specific page on android.com for a given ID, but this doesn't exist right? (And there's also the issue with ID prefixes not matching up).

Reply:
Would a good compromise to be make the link text `https://storage.googleapis.com/android-osv/<ID>.json` and the hyperlink target https://storage.googleapis.com/android-osv ?

In this vein of user experience, I previously found a very nice `index.html` file (I can't remember where) that I've been sticking in our public GCS buckets to make them imminently more browse able by humans, e.g. https://storage.googleapis.com/osv-vulnerabilities/index.html (you can grab it from this bucket). It also requires an `icons` folder, which you can also grab from https://storage.googleapis.com/osv-vulnerabilities/index.html?prefix=icons/

Reply:
> make the link text https://storage.googleapis.com/android-osv/<ID>.json and the hyperlink target https://storage.googleapis.com/android-osv ?

This works.

> index.html

Thanks! We'll look into throwing this onto our bucket.